app/src/main/java/com/hseun/lendy/auth/AuthRepository.kt

```
@@ -0,0 +1,95 @@
+        return try {
+            val response = authApi.refresh("Bearer $refreshToken")
+            if (response.isSuccessful) {
+                response.body()?.let {
+                    saveToken(it.accessToken, it.refreshToken)
+                    true
+                } ?: false
+            } else {
+                clearToken()
+                false
+            }
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)